pkgs/development/python-modules/pymusic-dl/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, click
+, prettytable
+, pycryptodome
+, pytest
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "pymusic-dl";
+  version = "3.0.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-A/cBv7TIZX3Vhua3HNwV5sbQmk/DrePm9qykV9X9Rko=";
+  };
+
+  propagatedBuildInputs = [
+    click
+    prettytable
+    pycryptodome
+    requests
+  ];
+
+  checkInputs = [
+    pytest
+  ];
+
+  meta = with lib; {
+    description = "Search and download music from multiple sources";
+    homepage = "https://github.com/0xHJK/music-dl";
+    license = licenses.mit;
```

Comment:
A maintainer is required. See https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md

Reply:
Fixed.  Many python packages lack maintainers property.  Therefore, I thought maybe python packages are maintained by a special python-maintainers group and does not require the maintainers property.